src/elements/ChoiceField/ChoiceField.styl

```diff
@@ -91,14 +106,16 @@
       position absolute
       top $space-small
       left $space-small
+      margin 4px
```

Comment:
To reiterate, hardcoding values is a bad idea. If a particular value comes up again and again, it should be made a token. Also working in `px` units is bad because it would break on resizing the screen.